modules/gapi/src/executor/gstreamingexecutor.cpp

```diff
@@ -1231,10 +1231,6 @@ void cv::gimpl::GStreamingExecutor::setSource(GRunArgs &&ins)
                                real_video_completion_cb);
     }
 
-    for (auto &&op : m_ops) {
-        op.isl_exec->handleNewStream();
-    }
```

Comment:
So where it is called now?

Reply:
https://github.com/opencv/opencv/blob/master/modules/gapi/src/executor/gstreamingexecutor.cpp#L1258